frontside/lib/views/home/home.dart

```diff
@@ -2,7 +2,7 @@ import 'package:flutter/material.dart';
 import 'package:frontside/parts/navbar.dart';
 
 class Home extends StatelessWidget {
-  static const String route = "/";
+  static const String route = "/home"; //default "/"
```

Comment:
Please change it back to `/`. It should be protected by a guard: https://pub.dev/packages/beamer#guards